R/drive_share.R

```diff
@@ -58,31 +65,59 @@ drive_share <- function(file, role = NULL, type = NULL, ..., verbose = TRUE) {
     )
   }
 
+  params <- list(...)
+  params[["role"]] <- role
+  params[["type"]] <- type
+  params[["fields"]] <- "*"
+  permissions_resource <- purrr::map(file$id,
+                                     drive_share_one,
+                                     params = params,
+                                     verbose = verbose)
+  file[["permissions_resource"]] <- NULL
+  file <- tibble::add_column(file, permissions_resource = permissions_resource, .after = 1)
```

Comment:
yes - and also if they just shared something, since sharing alone adds a column (currently, but now that I've typed it I'm not certain that makes sense...)